docs/content/en/docs/reference/clusterspec/baremetal.md

```diff
@@ -12,13 +12,577 @@ The following additional optional configuration can also be included:
+Refers to the Kubernetes object with Tinkerbell-specific configuration. See `TinkerbellDatacenterConfig Fields` below.
+
+### kubernetesVersion (required)
+The Kubernetes version you want to use for your cluster. Supported values: `1.22`, `1.21`, `1.20`
+
+### managementCluster
+Identifies the name of the management cluster.
+If this is a standalone cluster or if it were serving as the management cluster for other workload clusters, this will be the same as the cluster name.
+Bare Metal EKS Anywhere clusters do not yet support the creation of separate workload clusters.
+
+### workerNodeGroupConfigurations (required)
+This takes in a list of node groups that you can define for your workers.
+You may define one or more worker node groups.
+
+### workerNodeGroupConfigurations.count (required)
+Number of worker nodes
+
+### workerNodeGroupConfigurations.machineGroupRef (required)
+Refers to the Kubernetes object with Tinkerbell-specific configuration for your nodes. See `TinkerbellMachineConfig Fields` below.
+
+### workerNodeGroupConfigurations.name (required)
+Name of the worker node group (default: md-0)
+
+### workerNodeGroupConfigurations.taints
+A list of taints to apply to the nodes in the worker node group.
+
+Modifying the taints associated with a worker node group configuration will cause new nodes to be rolled-out, replacing the existing nodes associated with the configuration.
+
+At least one node group must not have `NoSchedule` or `NoExecute` taints applied to it.
+
+### workerNodeGroupConfigurations.labels
+A list of labels to apply to the nodes in the worker node group. This is in addition to the labels that
+EKS Anywhere will add by default.
+
+Modifying the labels associated with a worker node group configuration will cause new nodes to be rolled out, replacing
+the existing nodes associated with the configuration.
+
+## TinkerbellDatacenterConfig Fields
+
+### tinkerbellIP
+Required field to identify the IP address of the Tinkerbell service.
+Other TinkerbellDatacenterConfig fields are not yet supported.
+See [Artifacts]({{< relref "../artifacts/" >}}) for details.
+
+### osImageURL
+Optional field to replace the default operating system image.
+This field is useful if you want to provide a customized operating system image or simply host the standard image locally.
+See [Artifacts]({{< relref "../artifacts/" >}}) for details.
+
+### hookImagesURLPath
+Optional field to replace the HookOS image.
+This field is useful if you want to provide a customized HookOS image or simply host the standard image locally.
+
+## TinkerbellMachineConfig Fields
+In the example, there are `TinkerbellMachineConfig` sections for control plane (`my-cluster-name-cp`) and worker (`my-cluster-name`) machine groups.
+The following fields identify information needed to configure the nodes in each of those groups.
+>**_NOTE:_** Currently, you can only have one machine group for all machines in the control plane and one for all machines in the worker group.
```

Comment:
We have multi-node-group support for worker node groups

Reply:
Done.